tests/performance/common/common.go

```diff
@@ -317,3 +319,29 @@ func SendHttpRequest(method, reqApi string, body io.Reader) (error, *http.Respon
 	}
 	return nil, resp
 }
+
+// GetLatency calculates latency based on different percent
+func GetLatency(pods []types.FakePod) types.Latency {
```

Comment:
where is this func used ?

Reply:
Hey rohit, please find it here:)
https://github.com/kubeedge/kubeedge/blob/c005656e5ab2d363640fba804c8fca49e38f161e/tests/performance/hubtest/hubtest_test.go#L50